R/ui.R

```diff
@@ -81,7 +81,9 @@ include_scripts <- function(opt) {
     "js/reset-p-id-and-refresh-browser.js",
     "js/show-footer.js",
     "js/trigger-button.js",
-    "js/navigate-away.js")
+    "js/navigate-away.js",
+    if(opt$get_user_info) "js/get-user-info.js" else NULL
```

Comment:
I think this can be shortened to 

```r
if(opt$get_user_info) "js/get-user-info.js"
```

and the NULL will returned automatically if `opt$get_user_info` is `FALSE`

Reply:
Yup. My style can be a bit overly verbose - I quite like seeing these things explicitly for readability - but I agree this is not necessary and will remove.